syntax = "proto3";

package rec_sys;

message GetRecResultReq {
    int32 userId = 1;
    int32 addressId = 2;
    int32 touristType = 3;
    int32 priceSensitive = 4;
    repeated int32 likeType = 5;
    repeated int32 targetType = 6;
    repeated int32 attentionType = 7;
    bool update = 8;
    int32 limit = 9;
    repeated int32 likedSiteIdxList = 10;
}

message GetRecResultResp {
    repeated int32 siteIdxList = 1;
}

service RecSysService {
    rpc GetRecResult(GetRecResultReq) returns (GetRecResultResp);
}

